File: NirvanaSMS.Api.Java/src/main/java/org/nirvanasms/api/core/interfaces/IReport.java
package org.nirvanasms.api.core.interfaces;

import java.util.List;

import org.nirvanasms.api.core.base.UserBase;
import org.nirvanasms.api.core.entity.NumberReport;
import org.nirvanasms.api.core.utility.ProcessResult;

public interface IReport {
	public ProcessResult<List<NumberReport>> getReport(UserBase apiUser, String messageId);
}

File: NirvanaSMS.Api.CSharp/LibraryProject/Core/Interface/IReport.cs
using NirvanaSMS.Api.Core.Base;
using NirvanaSMS.Api.Core.Entity;
using NirvanaSMS.Api.Core.Utility;
using System.Collections.Generic;

namespace NirvanaSMS.Api.Core.Interface
{
    public interface IReport
    {
        ProcessResult<List<NumberReport>> GetReport(UserBase apiUser, string messageId);
    }
}
